vpt/tests/test_gaze_detector.py

```diff
@@ -0,0 +1,71 @@
+import sys
+# Test supposed to run as python -m unittest path/to/file.py
+# from the root of repository
+# If you know better way than using this sys.path thingy, tell me
+sys.path.append('.')
```

Comment:
Why is this needed?

Reply:
Well, uuuugh...  Yeah, I forgot to mention it. Oops... The problem is that it cannot find `vpt` module :(